src/modules/dashboard/application/use-cases/widgets/__tests__/get-system-health.use-case.spec.ts

```
@@ -0,0 +1,221 @@
+    it('should execute health checks in parallel', async () => {
+      const mockDbHealth = {
+        isHealthy: true,
+        responseTime: 25,
+      };
+      const mockRedisHealth = {
+        isHealthy: true,
+        responseTime: 5,
+      };
+
+      let dbCheckStarted = false;
+      let redisCheckStarted = false;
```

ESLint

[error] 'redisCheckStarted' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)